src/__tests__/shared/components/input/SelectWithLabelFormGroup.test.tsx

```diff
@@ -18,64 +20,58 @@ describe('select with label form group', () => {
         />,
       )
 
-      const label = wrapper.find(Label)
-      expect(label).toHaveLength(1)
-      expect(label.prop('htmlFor')).toEqual(`${expectedName}Select`)
-      expect(label.prop('text')).toEqual(expectedName)
+      expect(screen.getByText(expectedName)).toHaveAttribute('for', `${expectedName}Select`)
     })
 
     it('should render disabled is isDisable disabled is true', () => {
-      const expectedName = 'test'
-      const wrapper = shallow(
+      render(
         <SelectWithLabelFormGroup
           options={[{ value: 'value1', label: 'label1' }]}
-          name={expectedName}
+          name="test"
           label="test"
           isEditable={false}
           onChange={jest.fn()}
         />,
       )
 
-      const select = wrapper.find(Select)
-      expect(select).toHaveLength(1)
-      expect(select.prop('disabled')).toBeTruthy()
+      expect(screen.getByRole('combobox')).toBeDisabled()
     })
 
     it('should render the proper value', () => {
-      const expectedName = 'test'
-      const expectedDefaultSelected = [{ value: 'value', label: 'label' }]
-      const wrapper = shallow(
+      const expectedLabel = 'label'
+      render(
         <SelectWithLabelFormGroup
-          options={[{ value: 'value', label: 'label' }]}
-          name={expectedName}
+          options={[{ value: 'value', label: expectedLabel }]}
+          name="test"
           label="test"
-          defaultSelected={expectedDefaultSelected}
-          isEditable={false}
+          defaultSelected={[{ value: 'value', label: expectedLabel }]}
+          isEditable
           onChange={jest.fn()}
         />,
       )
 
-      const select = wrapper.find(Select)
-      expect(select).toHaveLength(1)
-      expect(select.prop('defaultSelected')).toEqual(expectedDefaultSelected)
+      expect(screen.getByRole('combobox')).toHaveValue(expectedLabel)
```

Comment:
Inspecting the DOM, there did not appear to be any way to identify the currently selected `value` of the field, only the `label` text.